### ISSUE TITLE
Penile candidiasis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6405]
**Preferred term label:**
Penile candidiasis

**Synonyms**
Penile thrush

**Definition (free text, please give PubMed ID)**
Overgrowth of candida albicans in the penis.
PMID: 27659805

**Parent term (use hpo.jax.org/app)**
Abnormality of the penis

**Comment by pnrobinson** [comment]
adding as HP:0033768, sorry for the delay
